package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
I tried ff merging and pushing directly to this repo using this PR and it works like a charm, thanks for the hint!